test/test_api_io.ml

```diff
@@ -0,0 +1,34 @@
+let test ~name ~body ~expected_errors =
+  ( name
+  , `Quick
+  , fun () ->
+      let json = Yojson.Safe.from_string body in
+      let result = Cs_api_io.get_graphql_errors json in
+      Alcotest.(check (list string)) name result expected_errors )
+
+let body_ok =
+  "{\"Not an error\":[{\"message\":\"OK \
+   found\",\"locations\":[{\"line\":3,\"column\":9}],\"path\":[\"createTrace\"]}],\"data\":{\"createTrace\":null}}"
+
+let body_error =
+  "{\"errors\":[{\"message\":\"No project with this ID was \
+   found\",\"locations\":[{\"line\":3,\"column\":9}],\"path\":[\"createTrace\"]}],\"data\":{\"createTrace\":null}}"
+
+let body_no_error_message =
+  "{\"errors\":[{\"locations\":[{\"line\":3,\"column\":9}],\"path\":[\"createTrace\"]}],\"data\":{\"createTrace\":null}}"
+
+let body_multiple_errors =
+  "{\"errors\":[{\"message\":\"No project with this ID was \
+   found\"},{\"message\":\"No project with this name was \
+   found\"}],\"data\":{\"createTrace\":null}}"
+
+let accumulator =
```

Comment:
I think this is the wrong name. It comes from the accumulator function we used to parse multipart form-data.